lib/resty/aes.lua

```diff
@@ -202,6 +209,17 @@ function _M.new(self, key, salt, _cipher, _hash, hash_rounds, iv_len)
          end
     end
 
+    if C.EVP_CIPHER_CTX_set_padding(encrypt_ctx, padding) == 0 then
+        return nil, "failed to set padding for encrypt context"
+    end
+
+    if C.EVP_CIPHER_CTX_set_padding(decrypt_ctx, padding) == 0 then
+        return nil, "failed to set padding for decrypt context"
+    end
+
+    ffi_gc(encrypt_ctx, C.EVP_CIPHER_CTX_free)
+    ffi_gc(decrypt_ctx, C.EVP_CIPHER_CTX_free)
```

Comment:
these two lines are useless since:
https://github.com/openresty/lua-resty-string/blob/a47d675e9405d3b3f34befb9550c526c6958aada/lib/resty/aes.lua#L126-L133

Reply:
fixed